library/Rules/PostalCode.php

```diff
@@ -53,7 +53,7 @@ class PostalCode extends Regex
         "FM" => "/^(\d{5})$/",
         "FO" => "/^(?:FO)*(\d{3})$/",
         "FR" => "/^(\d{5})$/",
-        "GB" => "/^(([A-Z]\d{2}[A-Z]{2})|([A-Z]\d{3}[A-Z]{2})|([A-Z]{2}\d{2}[A-Z]{2})|([A-Z]{2}\d{3}[A-Z]{2})|([A-Z]\d[A-Z]\d[A-Z]{2})|([A-Z]{2}\d[A-Z]\d[A-Z]{2})|(GIR0AA))$/",
+        "GB" => "/^([Gg][Ii][Rr] 0[Aa]{2})|((([A-Za-z][0-9]{1,2})|(([A-Za-z][A-Ha-hJ-Yj-y][0-9]{1,2})|(([AZa-z][0-9][A-Za-z])|([A-Za-z][A-Ha-hJ-Yj-y][0-9]?[A-Za-z])))) [0-9][A-Za-z]{2})$/",
```

Comment:
I know it is in the PDF you send, but `[AZa-z][0-9][A-Za-z]` doesn't make sense, `[A-Za-z][0-9][A-Za-z]`, are you sure I should keep this `[AZa-z][0-9][A-Za-z]`?